src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/model/prisonsapi/InmateDetail.kt

```diff
@@ -13,8 +15,8 @@ data class AssignedLivingUnit(
   val agencyName: String,
 )
 
-enum class InOutStatus {
-  IN,
-  OUT,
-  TRN,
+enum class InmateStatus(@JsonValue val value: String) {
+  IN("ACTIVE IN"),
```

Comment:
Are we ok to use the string value of `status` from NOMIS API response ? Because the `status` been formatted by this logic `this.status = format("%s %s", activeFlag ? "ACTIVE" : "INACTIVE", inOutStatus)` instead of constant or enumeration value.

Reply:
I don't think I follow. Where's that happening?